main.py

```diff
@@ -1,70 +1,93 @@
-## Mad libs generator
+#!/usr/bin/env python3
+# Mad libs generator
 
 import time
 
-tps = ['year', 'minute', 'second', 'hour', 'decade', 'week', 'day', 'month', 'decade', 'centur', 'eons', 'forever', 'millen']
+from helpers import *
```

Comment:
Organization! Just for me

Reply:
Ok! 

I follow the `helpers.py` purpose for sure. I think I'm gonna use that formatting cause it looks neat!
Is there a difference between `import x` and `From x import *` ? I see them both used in a few examples I've been looking at and I cant really tell.

I didn't see anything on stack overflow-- or maybe im looking up the wrong thing

Reply:
`import x` means 'import the module x'. If you want to use a function in that module, like `do()` you'd have to access it like `x.do()`. If you don't want to have to put the module name you can say `from x import do` and now you can just call `do()` without the identifier. `*` just means all the functions